injector/network_disruption.go

```diff
@@ -31,7 +31,7 @@ type linkOperation func([]string, string, string) error
 type networkDisruptionService struct {
 	ip       *net.IPNet
 	port     int
-	protocol network.Protocol
+	protocol v1.Protocol
```

Comment:
keep original type so we can improve the safeness of our internal type `protocol` by making it private.